lib/reporter.js

```diff
@@ -240,16 +250,17 @@ export default class Reporter {
   setRequestFunction (requestFunction) {
     this.request = requestFunction
   }
-}
 
-function isBundledFile (fileName) {
-  return fileName.indexOf(atom.getLoadSettings().resourcePath) === 0
-}
+  isBundledFile (fileName) {
+    return fileName.indexOf(this.resourcePath) === 0
```

Comment:
I don't think this ever worked on Windows - (or isTeletypeFile below) as `atom.getLoadSettings().resourcePath` is going to return an absolute Windows path, e.g. `C:\Users\damie\AppData\Local\Atom x64\app-dev\resources\app.asar` but we convert the filename into a linux style forward-slash one. 

Ditto `atom.packages.resolvePackagePath('teletype')`